### DISCUSSION TITLE
External link colour inside a button

### DISCUSSION THREAD

**Discussion by Fazle-Jahangir** [twilio-labs/paste#2592] [General] [answered]
Hi team, 

I would appreciate more clarity on the colour to go with when adding an external link inside a secondary button as shown in this screenshot (first image) included in the paste guidelines: https://paste.twilio.design/patterns/button-vs-anchor/#figma

This is not explored in the guidelines for buttons: https://paste.twilio.design/components/button/. We are currently implementing something similar with an external link added to the button but in black shown in the second screenshot (second image) instead of blue. Thanks for the help. 

![button-anchor-like-buttons](https://user-images.githubusercontent.com/100706574/183061615-c500ad3a-e05d-4dc1-883a-0fef89a3ddba.png)
<img width="718" alt="Screenshot 2022-08-05 at 11 34 54" src="https://user-images.githubusercontent.com/100706574/183061775-63348ebb-b171-4ef3-8630-78431cfcafdb.png">

**Comment by loreina** [comment, marked as answer]
Hi @Fazle-Jahangir,

Unfortunately, the colour of the secondary button isn't modifiable in Paste when adding an external link. 

The first link you shared in the Button vs. Anchor pattern is outdated. Thanks for catching that, our team will update the docs to reflect the actual secondary button design. Thank you!

The docs site also has some guidance on [buttons with link functionality](https://paste.twilio.design/components/button#buttons-with-link-functionality). Hope this helps!
